<launch>
	<node pkg="measure_node" type="measure_node" name="measure_node" output="screen"/>

    <node pkg="gocator_publisher" type="gocator_publisher" name="gocator_publisher" output="screen"/>

<!--     <node name="rviz" pkg="rviz" type="rviz" args="-d $(find gocator_publisher)/param/rviz/default.rviz" /> -->
</launch>